File: src/main/java/io/github/robertomike/exceptions/ParsingNotSupportedException.java
package io.github.robertomike.exceptions;

import org.springframework.http.HttpStatus;
import org.springframework.web.bind.annotation.ResponseStatus;

/**
 * Exception
 */
@ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
public class ParsingNotSupportedException extends ExceptionContract {
    /**
     * @param message used to specify error
     */
    public ParsingNotSupportedException(String message) {
        super(message);
    }
}

File: src/main/java/io/github/robertomike/inject_model/exceptions/ParsingNotSupportedException.kt
package io.github.robertomike.inject_model.exceptions

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.ResponseStatus

/**
 * Exception
 */
@ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
class ParsingNotSupportedException(message: String, e: Throwable? = null) : ExceptionContract(message, e)